packages/govern-contract-utils/contracts/bitmaps/BitmapLib.sol

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-License-Identifier:    GPL-3.0
+ */
+
+pragma solidity 0.6.8;
+
+library BitmapLib {
+    bytes32 constant internal empty = bytes32(0);
+    bytes32 constant internal allowAll = empty;
+    bytes32 constant internal denyAll = bytes32(uint256(-1));
+
+    function flip(bytes32 map, uint8 index) internal pure returns (bytes32) {
```

Comment:
Would be great to have an inline comment with an example of the inputs and outputs of each operation.